apps/dashboard/src/components/variable/types.ts

```diff
@@ -1,3 +1,12 @@
+/**
+ * Variable with optionally additional context
+ * depending on where the variable was inserted
+ */
+export type VariableWithContext = {
+  name: string;
+  aliasFor?: string | null;
```

Comment:
@ChmaraX, we already have the LiquidVariable struct in the FE - see [here](https://github.com/novuhq/novu/blob/1f62cdbd0af6cb74db22613cb4215e8a0f9d2170/apps/dashboard/src/components/variable/parseStepVariables.ts#L3). I suggest expanding that and work with that structure instead of introducing a new one.